fft_folder/plot_mauna_loa.ipynb

```diff
@@ -0,0 +1,87 @@
+      "text/plain": [
+       "<Figure size 1200x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from scipy.fft import fft, ifft\n",
+    "from scipy.signal import get_window\n",
+    "\n",
+    "# Load the JSON data with the correct path\n",
+    "data = pd.read_json('/workspaces/CP1-24-HW5/mauna-loa-data/flask_monthly.json')\n",
+    "\n",
+    "# Correct column names to match 'Year' and 'Month' from the JSON data\n",
```

Comment:
Good flow of correcting columns --> padding and windowing --> FFT --> Noise Removal --> invFFT as laid out in the task description.